packages/stream_chat_flutter/lib/src/message_input.dart

```diff
@@ -1064,22 +1068,22 @@ class MessageInputState extends State<MessageInput> {
                               );
                             },
                     ),
-                    IconButton(
-                      padding: const EdgeInsets.all(0),
-                      icon: StreamSvgIcon.record(
-                        color: _getIconColor(3),
-                      ),
-                      onPressed: attachmentLimitCrossed ||
-                              (_attachmentContainsFile &&
-                                  _attachments.isNotEmpty)
-                          ? null
-                          : () {
-                              pickFile(
-                                DefaultAttachmentTypes.video,
-                                camera: true,
-                              );
-                            },
-                    ),
+                    // IconButton(
```

Comment:
Are we keeping this for some reason?

Reply:
Just in case we add video or document sharing via chat in the future 👀